jira_integration/jira_integration.php

```diff
@@ -0,0 +1,111 @@
+<?php
+// TODO: this header should not be required...
+header('Content-Type: text/plain; charset=UTF-8');
+
+// Only watch the dev environment
+if ($_ENV['PANTHEON_ENVIRONMENT'] != 'dev') {
```

Comment:
We should die if we are being called in test/live (which shouldn't happen), but we do want to support multidev!